docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/home_landing/paragraph--cgov-card-external.html.twig

```diff
@@ -4,6 +4,12 @@
  # get the title, description, image and URL from the paragraph fields.
  #}
 
+{% set cardType = 'featureCard' %}
+
+{% if 'thumbnail' in _context.theme_hook_suggestions[2] %}
```

Comment:
How solid it this approach? Random array indexes could be problematic. The theme hooks can change. Is there anyway to get at the display mode?